de_de/demo_glossar.md

```diff
@@ -32,11 +32,11 @@ $db_table = "rex_glossar";
 $sql = rex_sql::factory();
 $sql->setDebug(false); //Ausgabe Query true oder false
 $query = "SELECT * FROM $db_table  ORDER BY Begriff ";
-$sql->setQuery($query, array($id));
+$rows = $sql->getArray($query);
```

Comment:
müsste hier nicht noch array_filter() durch?

Reply:
?

Reply:
wenn du getArray() benutzt und es keine Ergebnisse gibt, erhältst du normalerweise trotzdem ein leeres Multiarray. Das muss erst weggefiltert werden. `array_filter($sql->getArray($query))` - oder nicht?

Reply:
muss ich mal schauen...

Reply:
also eigentlcih wird überall das Ergebnis von getArray() in einer foreach Schelife genutzt.